about.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="about.css" />
+
+    <title>*About* R&B Music World Tour 2023</title>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    <header>
+        <!---------TOP BAR------------>
+      <div class="top-bar_header">
+        <div class="top-bar_header_container">
+          <a href="#"
+            ><img src="./assets/logos/facebook.png" alt="Facebook logo"
+          /></a>
+          <a href="#"
+            ><img src="./assets/logos/twitter.png" alt="Twitter logo"
+          /></a>
+          <a href="#">English</a>
+          <a href="#">My Page</a>
+          <a href="#">Logout</a>
+        </div>
+      </div>
+
+        <!---------NAVIGATION BAR---------->
+      <nav>
+        <div id="nav-bar" class="navbar">
+         <!------------TOUR LOGO------------->
+          <div class="tour-logo">
+            <a href="main.html">
+              <img src="./assets/logos/eye-orange.PNG" alt="eye icon" />
+              <div class="tour-logo_text">
+                <h4 class="tour-logo_up">R&B Music</h4>
+                <h3 class="tour-logo_mid">World Tour</h3>
+                <h4 class="tour-logo_bot">2023</h4>
+            </div>
+        </a>
+    </div>
+
+                <!--------NAV BAR LINKS------->
+                <ul class="nav_links">
+                  <li><a href="main.html">Home</a></li>
+                  <li><a href="#">About</a></li>
+                  <li><a href="main.html#program">Program</a></li>
+                  <li><a href="main.html#lineup">Lineup</a></li>
+                  <li><a href="main.html#sponsors_section">Sponsors</a></li>
+                  <li><a href="#">Tour 2023</a></li>
+                </ul>
+
+                <!--------BURGER MENU---------->
+                <button id ="hamb" class="nav-bar_menu" type="button">
+                  <span class="bar top"></span>
+                  <span class="bar mid"></span>
+                  <span class="bar bot"></span>
+                </button>
+              </div>
+            </nav>
```

Comment:
- [ ] **Optional**: About page, Mobile size:
- I would advise you to make the background transparent or the same color. It is part of the design requirements.
- Your page:
![Capture](https://github.com/rudicarrilloypr/capstone-module-1/assets/63972670/40b2a696-e7af-4c97-83c8-7f498e2e62d6)
- Original page




![Capture](https://github.com/rudicarrilloypr/capstone-module-1/assets/63972670/f5f137ba-7759-4980-8d64-a41ac1c95178)